extensions/amp-accordion/1.0/README.md

```diff
@@ -0,0 +1,641 @@
+  })();
+</script>
+```
+
+**collapse()**
+The `collapse` action collapses the sections of the `bento-accordion`. If a
+section is already collapsed, it stays collapsed. When called with no arguments,
+it collapses all sections of the accordion. To specify a section, add the
+`section` argument, and use its corresponding `id` as the value.
+
+```html
+<button id="button1">Collapse All Sections</button>
+<button id="button2">Collapse Section 1</button>
+<script>
+  (async () => {
+    const accordion = document.querySelector('#myAccordion');
+    await customElements.whenDefined('bento-accordion');
+    const api = await accordion.getApi();
+
+    // set up button actions
+    document.querySelector('#button1').onclick = () => api.collapse();
+    document.querySelector('#button2').onclick = () => api.collapse('section1');
+  })();
+</script>
+```
+
+##### Events
+
+The `bento-accordion` API allows you to register and respond to the following events:
+
+**expand**
+This event is triggered when an accordion section is expanded and is dispatched from the expanded section.
+
+See below for example.
+
+**collapse**
+This event is triggered when an accordion section is collapsed and is dispatched from the collapsed section.
+
+In the example below, `section 1` listens for the `expand` event and expands `section 2` when it is expanded. `section 2` listens for the `collapse` event and collapses `section 1` when it is collapsed.
+
+See below for example.
+
+```html
+<bento-accordion id="eventsAccordion" animate='true'>
+  <section id="section1">
+    <h2>Section 1</h2>
+    <div>Puppies are cute.</div>
+  </section>
+  <section id="section2">
+    <h2>Section 2</h2>
+    <div>Kittens are furry.</div>
+  </section>
+</bento-accordion>
+
+<script>
+  (async () => {
+    const accordion = document.querySelector('#eventsAccordion');
+    await customElements.whenDefined('bento-accordion');
+    const api = await accordion.getApi();
+
+    // when section 1 expands, section 2 also expands
+    // when section 2 collapses, section 1 also collapses
+    const section1 = document.querySelector('#section1');
+    const section2 = document.querySelector('#section2');
+    section1.addEventListener('expand', () => api.expand('section2'));
+    section2.addEventListener('collapse', () => api.collapse('section1'));
+  })();
+</script>
+```
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-accordion-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-accordion {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
```

Comment:
Let's add the other styles in `extensions/amp-accordion/1.0/amp-accordion.css` here

Reply:
all of them?

Reply:
I think so - many of them have border/padding styles and hiding (for non expanded sections) that will cause CLS if not available one way or another.

Reply:
added